native/source/mixpanel/detail/worker.cpp

```diff
@@ -190,6 +196,50 @@ namespace mixpanel
             }
         }
 
+        int Worker::parse_www_retry_after(nanowww::Response& response)
```

Comment:
We could fix that by adding e.g.

    inline const Headers * headers() const { return &hdr_; }

as an additional overload of the `headers` member function. Probably not worth it unless upstream is willing to accept patches, though.